Linux x86 build error
```
2022-06-14T20:13:06.9740785Z [ 14%] Building C object libs-native/System.Net.Security.Native/CMakeFiles/System.Net.Security.Native-Static.dir/pal_gssapi.c.o
2022-06-14T20:13:07.1174601Z /__w/1/s/src/native/libs/System.Net.Security.Native/pal_gssapi.c:162:9: error: macro is not used [-Werror,-Wunused-macros]
2022-06-14T20:13:07.1177258Z #define PER_FUNCTION_BLOCK(fn) \
2022-06-14T20:13:07.1178186Z         ^
2022-06-14T20:13:07.1265099Z 1 error generated.
```

--- a/src/coreclr/nativeaot/Runtime/windows/CoffNativeCodeManager.cpp
+++ b/src/coreclr/nativeaot/Runtime/windows/CoffNativeCodeManager.cpp
@@ -722,7 +722,7 @@ bool CoffNativeCodeManager::GetReturnAddressHijackInfo(MethodInfo *    pMethodIn
 #if defined(TARGET_ARM) || defined(TARGET_ARM64)
     flags = (GcInfoDecoderFlags)(flags | DECODE_HAS_TAILCALLS);
 #endif // TARGET_ARM || TARGET_ARM64
-    GcInfoDecoder decoder(GCInfoToken(p), DECODE_RETURN_KIND);
+    GcInfoDecoder decoder(GCInfoToken(p), flags);
 
     GCRefKind gcRefKind = GetGcRefKind(decoder.GetReturnKind());
 

